server/database/models/User.js

```diff
@@ -4,28 +4,33 @@ const jwt = require("jsonwebtoken");
 const config = require("../../config/config");
 const Thread = require("./Thread");
 
-const userSchema = new Schema(
-  {
-    name: {
-      type: String,
-      required: true
-    },
-    email: {
-      type: String,
-      required: true
-    },
-    password: {
-      type: String,
-      required: true
-    },
-    experience: {
-      type: Schema.Types.Mixed,
-      of: Number
-    },
-    assigned_threads: [{ type: Schema.Types.ObjectId, ref: Thread }]
+const userSchema = new Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  email: {
+    type: String,
+    required: true
   },
-  { timestamps: true }
-);
+  password: {
+    type: String,
+    required: true
+  },
+  experience: {
+    type: Schema.Types.Mixed,
+    of: Number
+  },
+  assigned_threads: [{ type: Schema.Types.ObjectId, ref: Thread }],
+  registeredDate: {
+    type: Date,
+    default: Date.now
```

Comment:
registeredDate is no longer a path on User as that's being store in the createdAt path which is automatically created with the timestamps: true option.

Reply:
I fixed this, it must have accepted the wrong changes when resolving conflicts locally. Did you notice anything else out of sync?